pkg/config/config.go

```diff
@@ -86,7 +95,11 @@ func GetSyncthingLogFile(namespace, name string) string {
 
 // GetUserHomeDir returns the OS home dir
 func GetUserHomeDir() string {
-	if v, ok := os.LookupEnv("OKTETO_HOME"); ok {
+	if v, ok := os.LookupEnv("OKTETO_USERHOME"); ok {
```

Comment:
what is the use case for `OKTETO_USERHOME`?

Reply:
when we can't determine the path to the $HOME directory. that was the reason why we put OKTETO_HOME in the first place.

Reply:
but why would a user set `OKTETO_USERHOME` instead of `OKTETO_HOME`?

Reply:
we need to know where's the $HOME directory to be able to write the ssh configuration and kubeconfig files